target/linux/ramips/image/mt76x8.mk

```diff
@@ -58,6 +58,13 @@ define Device/hc5661a
 endef
 TARGET_DEVICES += hc5661a
 
+define Device/hc5861b
+  DTS := HC5861B
+  IMAGE_SIZE := $(ralink_default_fw_size_16M)
```

Comment:
ralink_default_fw_size_16M is 16121856 byte == 0xF60000. But your firmware partition has a size of 0xf70000 bytes. Please use `15808k` here.